android/titanium/build.gradle

```diff
@@ -261,21 +261,21 @@ dependencies {
 	implementation 'androidx.vectordrawable:vectordrawable:1.1.0'
 	implementation 'androidx.vectordrawable:vectordrawable-animated:1.1.0'
 	implementation 'androidx.viewpager:viewpager:1.0.0'
-	compileOnly 'androidx.annotation:annotation:1.3.0'
+	implementation 'androidx.annotation:annotation:1.5.0'
 
 	// Google's "Material Components" themed UI library.
 	implementation "com.google.android.material:material:${project.ext.tiMaterialLibVersion}"
 
 	// The Google Play Services libraries are only used by Titanium's geolocation feature.
 	// We link to them dynamically at runtime. So, they can be safely excluded when in the app project.
 	implementation "com.google.android.gms:play-services-base:${project.ext.tiPlayServicesBaseLibVersion}"
-	implementation 'com.google.android.gms:play-services-location:19.0.1'
+	implementation 'com.google.android.gms:play-services-location:20.0.0'
```

Comment:
Should this be updated along with ti.playservices?

Reply:
yes, that would be great to have this https://github.com/tidev/ti.playservices/pull/230 in the official package too.